viewer.py

```diff
@@ -258,13 +269,14 @@ def pro_init():
     """Function to handle first-run on Screenly Pro"""
     is_pro_init = path.isfile(path.join(settings.get_configdir(), 'not_initialized'))
 
-    if is_pro_init:
-        logging.debug('Detected Pro initiation cycle.')
-        load_browser(url=HOME + INTRO)
-    else:
+    if not is_pro_init:
```

Comment:
The code below this line is not needed in the OSE version. In the OSE version this is always False, and the Pro version uses different code here.